applications/Profond.md

```diff
@@ -0,0 +1,154 @@
+
+We helped grow the NEAR NFT ecosystem by contributing to NFT with open-source projects such as NFT smart contract standard, marketplace smart contract, vesting smart contract, and so on as you can see on the project’s repo: [https://github.com/parasHQ/](https://github.com/parasHQ/)
+
+Here’s our recent works:
+
+Paras Launchpad: [https://launchpad.paras.id/](https://launchpad.paras.id/) - NFT Launchpad in Astar using ink!
+
+Arkana Raffler: [https://arkana.gg/](https://arkana.gg/)
+
+Pipapo Ticketing System: [https://pipapo.io/](https://pipapo.io/)
+
+### Team Code Repos
+
+- [https://github.com/emarai](https://github.com/emarai)
+- [https://github.com/ekkirinaldi](https://github.com/orgs/profondHQ/people/ekkirinaldi)
+- [https://github.com/bagasprakasa](https://github.com/bagasprakasa)
+- [https://github.com/internnos](https://github.com/internnos)
+- [https://github.com/adeyusupp](https://github.com/adeyusupp)
+
+### Team LinkedIn Profiles (if available)
+
+- [https://www.linkedin.com/in/adhigunamahendra/](https://www.linkedin.com/in/adhigunamahendra/)
+- [https://www.linkedin.com/in/bagasprakasa/](https://www.linkedin.com/in/bagasprakasa/)
+- [https://www.linkedin.com/in/irfianto/](https://www.linkedin.com/in/irfianto/)
+- [https://www.linkedin.com/in/amajidsinar/](https://www.linkedin.com/in/amajidsinar/)
+- [https://www.linkedin.com/in/ekkirinaldi/](https://www.linkedin.com/in/ekkirinaldi/)
+- [https://www.linkedin.com/in/vincentsalaka/](https://www.linkedin.com/in/vincentsalaka/)
+- [https://www.linkedin.com/in/adeyusupp/](https://www.linkedin.com/in/adeyusupp/)
+
+## Development Status :open_book:
+
+PoC → Here’s our current MVP in Sui [http://console.profond.ai/](http://console.profond.ai/)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
+- **Full-Time Equivalent (FTE):** 2 FTE
+- **Total Costs:** $10.000
+
+### Milestone 1 — No code smart contract builder
+
+- **Estimated duration:** 1 month
+- **FTE:** 2
+- **Costs:** 4,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide documentation page on how to create NFT (PSP34) and FT (PSP22) smart contract with our interface. API documentation for project settings will also be provided.  |
+| 0c. | Testing and Testing Guide | All functions will be covered with full integration test, testing guide will also be provided. |
+| 0d. | Article | We will publish an article that explains the no code builder |
+| 1. | No Code NFT Creation (ink!) | Allows users to create NFT collection smart contract (PSP34) using ink! in one of WASM supported Parachains. |
+| 2. | No Code Fungible Token Creation (ink!) | Allows users to create Fungible Token smart contract (PSP22) using ink! in one of WASM supported Parachains. |
```

Comment:
Cool! Can you expand the technical details a bit in the deliverables? What technologies will you use to build this?

Reply:
Yep, I have updated the technical details inside the `Specification`.